Source/utils/stdcompat/cstddef.hpp

```diff
@@ -1,16 +1,14 @@
 #pragma once
 
 #ifdef __has_include
-#if __has_include(<cstddef>)
 #include <cstddef>
```

Comment:
Please move outside of `__has_include`

Reply:
did you mean move "#include <cstddef>" above "#ifdef __has_include"

Reply:
Yeah